usecases/monitoring/prometheus.go

```diff
@@ -230,6 +231,10 @@ func newPrometheusMetrics() *PrometheusMetrics {
 			Name: "backup_store_data_transferred",
 			Help: "Total number of bytes transferred during a backup store",
 		}, []string{"backend_name", "class_name"}),
+		DimensionSum: promauto.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "dimension_sum",
+			Help: "Total dimensions in a shard",
+		}, []string{"class_name", "shard_name"}),
```

Comment:
one comment on the naming, wouldn't it be better to find if the metric would be called: `vector_dimensions_sum`?